src/docs/patterns/navigation.mdx

```diff
@@ -22,20 +37,23 @@ Navigation is the way in which users access the different areas of your applicat
 
 ## Fixed Navigation
 
-The most basic navigation panel is fixed in place and always visible on the left side of the application. Note: a fixed navigation panel is not suitable when going to smaller resolutions or mobile devices.
+The most basic navigation panel is fixed in place and always visible on the left side of the application The navigation can be a fixed list of icons with labels or a fixed panel that is open and revealing multiple navigation options Due to internationalization and accessibility standards PX Blue recommends the use of an open panel displaying the navigation options Another benefit of using a fixed navigation is the ability to use expandable tree navigation as well as more area for product branding. Note: A fixed navigation panel is not suitable when the application is viewed on smaller resolutions or mobile devices
 
 ## Collapsible Navigation
 
 A more common implementation of the navigation panel is a collapsible panel that can be opened and closed via the menu icon in the top left corner of the application. This implementation can have two different variations: content-resize or content-cover.
```

Comment:
Do we want to mention anything about hover behavior here, or are we anticipating taking that out of the component library?

Reply:
If I can get an animation to work it is def worth putting in. It may be difficult to describe the desired behavior without some type of animation

Reply:
I'm just talking about how (in the currently implementation) the persistent drawer will open temporarily when you hover over it. It was mentioned in a design review a while back that we might want to remove that behavior and just have click to open/close. Not sure where we landed on that.